app/assets/stylesheets/components/banner.scss

```diff
@@ -1,5 +1,5 @@
-.banner-temporary-message-without-action {
-  background: govuk-colour("blue");
+.banner {
+  background: govuk-colour("red");
```

Comment:
Please see main comment, I believe we should not be making design decisions ourselves given we have a UCD (Ux/Content/Design) team member